playbooks/de-populate-consul.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: "Update keys in consul"
```

Comment:
I think this playbook probably should delete existing configurations before updating things. I think it's better to clean it up every time than it is to have cruft in Consul.